main/acle.md

```diff
@@ -2535,6 +2537,15 @@ following:
 versioning mechanism described in this section is supported by the
 compiler and it is enabled.
 
+`__FUNCTION_MULTI_VERSIONING_SUPPORT_LEVEL` is defined to the currently supported
+version of the ACLE. The value and the format are the same as the `__ARM_ACLE`.
+
+For example could be implemted as:
```

Comment:
Better phrasing: "For example, it can be implemented as:"

Reply:
done